modules/installation-mirror-repository.adoc

```diff
@@ -86,6 +90,16 @@ $ PRODUCT_REPO='openshift'
 ----
 endif::[]
 
+.. Generate the certificate:
```

Comment:
This doc is not talking about how to set up registry, mirror registry is out of this doc's scope, so we do not need this.

Reply:
addressed